lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1094,6 +1160,11 @@ estimateTxSize skeleton =
     sizeOf_UInt = 5
     sizeOf_LargeUInt = 9
 
+    -- A CBOR Int which is less than 23 in value fits on a single byte. Beyond,
+    -- the first byte is used to encode the number of bytes necessary to encode
+    -- the number, followed by the number itself.
+    sizeOf_LargeInt = 9
```

Comment:
Would it be better to call this `sizeOf_Int64`? Because that's the max size for minting.

Reply:
Addressed in 51d5f9371.

Reply:
https://github.com/input-output-hk/cardano-wallet/pull/2741/files#diff-6218e625cd8c0f10e842ef1c10eb04bebf307ccecad42038e3b12b287a763690R1184